appinfo/routes.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+/**
+ * SPDX-FileCopyrightText: 2013 Nextcloud GmbH and Nextcloud contributors
```

Comment:
Need to double check if that is OK, or if we need to have a different copyright header since 2013 was before Nextcloud

Reply:
There is copyright from first commit: https://github.com/nextcloud/notes/commit/1c4ae20dad288b5691e87a1fabb5e944071ca782#diff-6301937af1cc1575e9e18ecd27c4a51f0d3cf9ce12e59a14709f626dfa3ef952R2-R6

Reply:
Maybe @AndyScherzinger can easily tell if we can attribute that as `Nextcloud GmbH and Nextcloud contributors`or if we need to keep Bernhard explicitly in there

Reply:
I'd say keep him explicitly there. Given code existing before Nextcloud has been founded, Nextcloud cannot be a Copyright holder in any way.
So for anything oder than 2016, best to keep the original copyright holder Infos in additionally.